C4_Sequence.puml

```diff
@@ -365,55 +375,20 @@ end box
   Boundary($alias, $label, "", $allTags, $link)
 !endprocedure
 
-
-' Index
-' ##################################
-
-!$lastIndex = 0
-!$index = 1
-
-!procedure increment($offset=1)
-  !$lastIndex = $index
-  !$index = $index + $offset
-!endprocedure
-
-!procedure setIndex($new_index)
-  !$lastIndex = $index
-  !$index = $new_index
-!endprocedure
-
-!function Index($offset=1)
-  !$lastIndex = $index
-  !$index = $lastIndex + $offset
-  !return $lastIndex
-!endfunction
-
-!function LastIndex()
-  !return $lastIndex
-!endfunction
-
-!function SetIndex($new_index, $offset=1)
-  !$lastIndex = $new_index
-  !$index = $new_index + $offset
-  !return $lastIndex
-!endfunction
-
 ' Relationship (redefinition)
 ' ##################################
 
-!unquoted procedure Rel_($e_index, $alias1, $alias2, $label, $direction)
-$getRel($direction, $alias1, $alias2, $e_index + ": " + $label, "", "", "", "", "")
-!endprocedure
-!unquoted procedure Rel_($e_index, $alias1, $alias2, $label, $techn, $direction)
-$getRel($direction, $alias1, $alias2, $e_index + ": " + $label, $techn, "", "", "", "")
-!endprocedure
+' only Rel is supported in sequence diagram
```

Comment:
I'm slightly concerned that removing various functions will break diagrams for users when these changes get released to `plantuml-stdlib`.

If at all possible, we should leave the functions in for backward compatibility (possibly refactored to align with the new paradigm) and only remove them from the docs.

Alternatively we might need to start thinking about how we can somehow warn the user that specific features/functions are being deprecated to allow a fair time before removal.

Reply:
if you mean removed `increment($offset=1) .....` this is only moved in C4.puml

If you mean removed `Rel_()` this was added in the new introduced C4_Sequence.puml last month (which was never released) therefore chance of a bad side effect is really low and the cleaning advantage is bigger